sdk/servicebus/azservicebus/ci.yml

```diff
@@ -0,0 +1,17 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/data/aztables
+
+pr:
+  paths:
+    include:
+    - sdk/servicebus/azservicebus
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'servicebus'
+    RunTests: true
```

Comment:
I believe this parameter is no longer necessary do to @seankane-msft's work.

Reply:
I believe you are a first class liar. Removed (because of trust), then added back (because of lies)

Reply:
He's more of a second-class liar. This will be unnecessary once I get #15167 merged